Formula/telegraf.rb

```diff
@@ -1,9 +1,10 @@
 class Telegraf < Formula
-  desc "Server-level metric gathering agent for InfluxDB"
-  homepage "https://www.influxdata.com/"
+  desc "Plugin-driven server agent for collecting & reporting metrics"
+  homepage "https://www.influxdata.com/time-series-platform/telegraf/"
   url "https://github.com/influxdata/telegraf/archive/v1.24.2.tar.gz"
   sha256 "e0b4bc65dc46a87b14a164c1226f6cee15fabfff14f355344bede3ef3e585925"
   license "MIT"
+  revision 1
```

Comment:
I don't think we need to force a reinstall for all users.
```suggestion
```

Reply:
Based on: https://docs.brew.sh/Formula-Cookbook#formulae-revisions

Reply:
> to either ensure formulae continue to function as expected or to close a security issue.

Neither is really applicable here

Reply:
What about

> In Homebrew we sometimes accept formulae updates that don’t include a version bump. These include resource updates, new patches or fixing a security issue with a formula.

Reply:
What about it? It doesn't say all those cases need a revision bump, it's just setting the scene.

Reply:
Okay thanks, I must have misread then.